eirslett/storybook-builder-vite
# Move to yarn, fix examples

As discussed in https://github.com/eirslett/storybook-builder-vite/pull/26, this moves us from npm workspaces to yarn workspaces, which seems to be more reliable and also matches up with what the main storybook monorepo uses.

This also adds scripts to start and build storybook in the examples, and fixes them up so they work correctly.

To test:

`yarn install` and then in each of the three example directories, run:

```
yarn storybook
```

Verify the build succeeds and the stories load correctly in the browser.

```
yarn build-storybook

npx http-server <output directory>
```

Verify the build succeeds and the stories load correctly in the browser.
